conversions/c_atoi_str_to_integer.c

```diff
@@ -0,0 +1,85 @@
+/**
+ * \file
+ * \brief Recoding the original atoi function in stdlib.h 
+<<<<<<< HEAD
```

Comment:
merge conflict here not resolved properly.

Reply:
yes it was